src/converters/notion/index.ts

```diff
@@ -0,0 +1,232 @@
+    const newValues = node.children?.map(child => child.name) ?? []
+    const newValueCount = newValues.length
+
+    return {
+        ...attributesMap,
+        [node.name]: {
+            ...oldAttribute,
+            count: oldAttribute.count + newValueCount,
+            values: [... new Set(oldAttribute.values.concat(newValues))],
+        }
+    }
+}
+
+const mergeAttributes = (a: AttributesMap, b: AttributesMap): AttributesMap => {
+    return Object.values(b).reduce((acc, curr) => {
+        const old = acc[curr.name]
+
+        if(!old){
+            return {
+               ...acc,
+               [curr.name]: curr 
+            }
+        }
+
+       
+        const newPatch = {
+            count: old.count + curr.count,
+            values: [...new Set(old.values.concat(curr.values))],
+        }
+
+        return {
+            ...acc,
+            [curr.name]: {
+                ...old,
+                ...newPatch
+            }
+        }
+    }, a)
+}
+
+const createAttributesMap = (nodes: TanaIntermediateNode[]): AttributesMap => {
+    const attributesMap = nodes.reduce((attributesAcc, node) => {
+        if(isField(node)){
+            attributesAcc = updateAttribute(attributesAcc, node)
+        }
+
+        const childrenAttributes = node.children ? createAttributesMap(node.children) : {}
+        return mergeAttributes(attributesAcc, childrenAttributes)
+    }, {} as AttributesMap)
+
+    return attributesMap
+}
+
+const createAttributes = (nodes: TanaIntermediateNode[]): TanaIntermediateAttribute[] => Object.values(createAttributesMap(nodes))
+
+const normalizeNodes = (nodes: TanaIntermediateNode[]) => {
+    return nodes.reduce((acc, curr) => {
+        acc[curr.name] = curr
+        return acc
+    }, {} as Record<TanaIntermediateAttribute['name'], TanaIntermediateNode>)
+}
+
+const getNameOutOfLink = (link: string) => {
+    return decodeURIComponent(link.replace(/%20[0-9a-f]{32}|\.md$/g, "")).split("/").at(-1) 
+}
+
+const fixPageLinks = (nodeMap: Record<TanaIntermediateAttribute['name'], TanaIntermediateNode>, nodes: TanaIntermediateNode[]): TanaIntermediateNode[] => {
+    return nodes.map((node) => {
+        const refs: string[] = []
+        const newName = node.name.split(',').map((potentialLink) => {
+            if(!isPageLink(potentialLink)){
+                return potentialLink
+            }
+
+            const linkName = getNameOutOfLink(potentialLink)
+            
+            if(!linkName || !nodeMap[linkName]){
+                return potentialLink
+            }
+
+            const refFromLink = nodeMap[linkName].uid
+
+            refs.push(refFromLink)
+            return `[[${refFromLink}]]`
+        }).join(',')
+
+        const newChildren =  node.children ? fixPageLinks(nodeMap, node.children) : undefined
+
+        return {
+            ...node,
+            name: newName,
+            refs,
+           children: newChildren 
+        }
+    })
+}
```

Comment:
After nodes are created, we can go through an find an links that can be replaced with node references.